pages/_app.tsx

```diff
@@ -1,12 +1,13 @@
-/* istanbul ignore file */
```

Comment:
This comment should remain in the file since we can't test this file on it's own.